mLRS/CommonTx/cli.h

```diff
@@ -351,6 +361,10 @@ void tTxCli::Init(tSerialBase* const _comport)
     task_pending = TX_TASK_NONE;
     task_value = 0;
 
+    output_pending = CLI_OUTPUT_NONE;
+    output_index = 0;
+    output_flag = 0;
```

Comment:
while a long due leftover, I think we could use the opportunity to also add an enum for the flags ... was ugly to read so far

Reply:
Since this is unrelated and you didn't mention what you would want to call the enums, I think I'll pass on this and let you fix it later as you see fit, lest this require another round of changes due to my likely poorly chosen names.